src/lib/server.ts

```diff
@@ -399,6 +413,29 @@ export interface Logger {
   debug(message: string): void;
 }
 
+export class ZwavejsRemoteController extends EventEmitter {
```

Comment:
Can we add a JS Doc why this class exists.

Reply:
Also, since it controls the server, should it be called something like `ZWaveJSServerRemoteController` ?